components/brave_prochlo/BUILD.gn

```diff
@@ -13,6 +13,7 @@ source_set("brave_prochlo") {
 
   public_deps = [
     ":prochlo_proto",
+    "//base:base",
```

Comment:
do we really need adding `base`?

Reply:
@iefremov - I get this error if we don't:

```
> gn check src/out/Component //brave/*
ERROR at //brave/components/brave_prochlo/brave_prochlo_message.h:11:11: Can't include this header from here.
#include "base/time/time.h"
          ^---------------
The target:
  //brave/components/brave_prochlo:brave_prochlo
is including a file from the target:
  //base:base

ERROR at //brave/components/brave_prochlo/brave_prochlo_crypto.h:11:11: Can't include this header from here.
#include "base/macros.h"
          ^------------
The target:
  //brave/components/brave_prochlo:brave_prochlo
is including a file from the target:
  //base:base
```